connect-inject/lifecycle_sidecar_test.go

```diff
@@ -1,12 +1,20 @@
 package connectinject
 
 import (
+	"testing"
+
 	"github.com/hashicorp/go-hclog"
 	"github.com/stretchr/testify/require"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"testing"
+)
+
+const (
```

Comment:
optional: if you made this a `var` you could make the `resource.MustParse` call here, e.g.
```go
var (
	hi = resource.MustParse("10m"),
)
```